rir/src/compiler/test/PirCheck.cpp

```diff
@@ -135,9 +135,9 @@ static bool testOneNot(ClosureVersion* f) {
     return numNots == 1;
 }
 
-static bool testLdFunInFirstBB(ClosureVersion* f) {
+static bool testLdVarInFirstBB(ClosureVersion* f) {
     for (auto instruction : *f->entry) {
-        if (LdFun::Cast(instruction)) {
+        if (LdVar::Cast(instruction)) {
```

Comment:
I am afraid that this is not a very solid check, since it will trigger for any ldvar (which there's almost guaranteed to be one). Can you do sth like `testLdVarCInFirstBB` which tests for `ldvar "c"`